scene/2d/gpu_particles_2d.cpp

```diff
@@ -912,6 +916,7 @@ GPUParticles2D::GPUParticles2D() {
 	one_shot = false; // Needed so that set_emitting doesn't access uninitialized values
 	set_emitting(true);
 	set_one_shot(false);
+	seed = 0;
```

Comment:
GPUParticles3D calls `set_seed(0)` which is why your other PR doesn't trigger this same issue. 

```suggestion
	set_seed(0);
```

This PR should also use `set_seed(0)` so that the value is passed to the rendering server

Reply:
Sounds good.  I will go update this.